alpine-php/k8s/etc/nginx/apps/drupal/cron_allowed_hosts.conf

```diff
@@ -0,0 +1,10 @@
+# -*- mode: nginx; mode:autopair; mode: flyspell-prog; ispell-local-dictionary: "american" -*-
+### Configuration file for specifying which hosts can invoke Drupal's
+### cron. This only applies if you're not using drush to run cron.
+
+geo $not_allowed_cron {
+    default 1;
+    ## Add your set of hosts.
+    127.0.0.1 0; # allow the localhost
+    192.168.1.0/24 0; # allow on an internal network
```

Comment:
Should this list also have 10.0.0.0/8 and 172.16.0.0/20?
Should 192.168.1.0/24 be changed to 192.168.0.0/16?

Reply:
Yes, did I drop those? D'oh.

Reply:
Wait, what? The config you've reviewed and the config I have in my local repo are not the same.

Reply:
Oh, I see. Nay, we don't actually use this config file at all, it's just part of the default set.